hibernate-orm-modules/src/test/resources/arquillian.xml

```diff
@@ -23,14 +23,9 @@
         <container qualifier="container.active-1" mode="suite" default="true">
             <configuration>
                 <property name="jbossHome">${wildFlyInstallDir}</property>
-                <!-- For Remote debugging of Wildfly add the following line to the 'javaVmArguments' section below: -->
-                <!-- -Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=y-->
-                <!-- DO NOT add comments within the property section below as the Arquillian parser can't deal with it -->
```

Comment:
ok for the change but could you keep the warning about the comments?

Reply:
I will just remove the commit